### PR TITLE
fix: setbit should return 0 on an empty instance

### PR DESCRIPTION
**Pull request by liuwenzhuang**
setbit return the original bit value stored at offset, so `setbit key 10 1` should return 0.

### CODE REVIEW

File: docs/manual/pubsub.md [hunk 1]
```diff
@@ -77,18 +77,12 @@ environment (test, staging, production...).
...
-subscribe
-$6
-second
-:2
+1) "subscribe"
+2) "first"
+3) (integer) 1
+1) "subscribe"
+2) "second"
+3) (integer) 2
```
Comment (itamarhaber) [inline docs/manual/pubsub.md:85]: Please revert - this is the wire protocol format, not redis-cli.

File: docs/manual/pubsub.md [hunk 2]
```diff
@@ -101,32 +95,22 @@ against the channel named `second`:
...
-*3
-$7
-message
-$6
-second
-$5
-Hello
+1) "message"
+2) "second"
+3) "Hello"
```
Comment (itamarhaber) [inline docs/manual/pubsub.md:100]: Same comment

File: docs/manual/pubsub.md [hunk 3]
```diff
@@ -101,32 +95,22 @@ against the channel named `second`:
...
-unsubscribe
-$5
-first
-:0
+1) "unsubscribe"
+2) "second"
+3) (integer) 1
+1) "unsubscribe"
+2) "first"
+3) (integer) 0
```
Comment (itamarhaber) [inline docs/manual/pubsub.md:113]: Same comment

**Review by itamarhaber — CHANGES_REQUESTED** [review]
Hello @liuwenzhuang - thanks for catching that erroneous example.

Please revert all the changes you've made to the examples of Pub/Sub's wire protocol.

**Comment by zuiderkwast** [comment]
> setbit return the original bit value stored at offset, so `setbit key 10 1` should return 0.

We don't know what was stored in the key before the example command was called. It may be 1.

For the other changes, they are just wrong (as @itamarhaber pointed out) so I think this PR can be closed.

**Comment by itamarhaber** [comment]
I still think the example is misleading, in the sense that on an empty instance, it should return 0.

**Comment by zuiderkwast** [comment]
Fair enoguh.

**Comment by liuwenzhuang** [comment]
> Hello @liuwenzhuang - thanks for catching that erroneous example.
> 
> Please revert all the changes you've made to the examples of Pub/Sub's wire protocol.

Done.

**Review by itamarhaber — APPROVED** [review]

**Pull request merged**